### DISCUSSION TITLE
File Line Numbers

### DISCUSSION THREAD

**Discussion by isadon** [magit/magit#4809] [Ideas]
Does magit support showing line numbers as per below? So line numbers that match the lines which were in the old file and line numbers based on the changes as they are in the current file.

<img width="669" alt="Screenshot 2022-11-04 at 2 40 57 PM" src="https://user-images.githubusercontent.com/1881038/200051804-a1e2b58f-52b6-40cd-98f7-b9a22213e3b4.png">

**Comment by kyleam** [comment]
> Does magit support showing line numbers as per below?

No, not that I'm aware of.

**Comment by isadon** [comment]
Feature Request please?
